docs/CONFIGURATION.md

```diff
@@ -282,15 +288,7 @@ eksctl create iamserviceaccount \
     --role-only
 ```
 
-See [Pod-Level identity](https://github.com/awslabs/mountpoint-s3-csi-driver/tree/main/examples/kubernetes/static_provisioning/pod_level_identity.yaml)
-examples for how to set up Pod-Level identity with IRSA.
-
-##### [Optional] Validate the account was successfully created
```

Comment:
I think this section is useful, it'd be better if we say “check if you have the right IAM Role ARN with `eks.amazonaws.com/role-arn` annotation.”

Reply:
I'm not really sure what you mean. The problem I have with this section is that we aren't dealing with service accounts at this point, but I'm open to suggestions on where this can be useful.

Reply:
I'm wondering if it makes more sense to include this when working on a troubleshooting document, which I think @muddyfish mentioned adding recently?

Reply:
I included the troubleshooting stuff I was talking about in this document in the STS section at the bottom.

Reply:
But this bit is where we're creating the service account, or at least the line above this one is.

Reply:
>this bit is where we're creating the service account, or at least the line above this one is

The service account isn't created in this documentation, right? (Otherwise my changes really don't make sense). 

What I understood is that we create IAM roles that will later be used for service accounts, when created either when installing the CSI driver or when setting up the namespace for the pod.

Reply:
@muddyfish can you check the thread above? I'd like to try and get this PR merged this week.